source/about_jena/citing.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Citing Jena
+---
+
+[comment]: <> (https://markmail.org/thread/k2hwm7m4c4dlo6ff)
```

Comment:
Can we use a `lists.apache.org` permalink instead?

Reply:
Force of habit. Had a look at lists.apache.org and liked the interface! Will try to start using that now. Thanks!